crates/bevy_ecs/src/schedule/runner/single_threaded.rs

```diff
@@ -0,0 +1,303 @@
+use crate::{
+    schedule::{Runner, RunnerApplyBuffers, SystemRunner},
+    world::World,
+};
+
+#[cfg(feature = "trace")]
+use bevy_utils::tracing::Instrument;
+use fixedbitset::FixedBitSet;
+
+/// Runs systems on a single thread.
```

Comment:
This needs to mention that the order still follows existing rules, and is not guaranteed to be stable.